src/course/syllabus/projects/in-house-design/schedule.md

```diff
@@ -51,50 +51,54 @@ schedule:
       end: 11:00
       type: presentation
     - name: Wireframing
-      start: 10:40
+      start: 11:00
       end: 13:00
       type: project
-    - name: Introduction to project roles
+    - name: Usability testing
       start: 14:00
-      end: 14:10
+      end: 16:00
+      url: https://docs.google.com/presentation/d/10VxYiFBmwLevEH3V6AP74ibf_JRt51Gd-295TICrWQU/edit?usp=sharing
+    - name: Synthesise Feedback
+      start: 16:00
+      end: 17:00
+    - name: Introduction to project roles
+      start: 17:00
+      end: 17:10
       url: /course/handbook/project-team
       type: presentation
     - name: Choose roles
-      start: 14:10
-      end: 14:20
-    - name: Technical spikes
-      start: 14:20
-      end: 16:20
-      url: ../spikes
-    - name: Spike presentation prep
-      start: 16:20
-      end: 16:50
-    - name: Spike presentations
-      start: 16:50
+      start: 17:10
+      end: 17:20
+    - name: Build sprint presentation
```

Comment:
This could go before Sprint Planning on Friday morning as that's kinda when they move from Design -> Build. A little more time can be given to synthesis and this afternoon might feel a little more fluid.